packages/side-nav/src/vaadin-side-nav-children-controller.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright (c) 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+
+/**
+ * A controller to manage the item content children slot.
+ */
+export class ChildrenController extends SlotController {
+  constructor(host) {
+    super(host, 'children', null, { observe: true, multiple: true });
```

Comment:
To be able to reuse this controller for the side-nav, I think we need to make the slot name overridable.

Reply:
Good point, thanks. I forgot that we use the default slot.

Reply:
Changed.